mdsuite/visualizer/d2_data_visualization.py

```diff
@@ -40,19 +40,22 @@ class DataVisualizer2D:
     Visualizer for two-dimensional data.
     """
 
-    def __init__(self, title: str):
+    def __init__(self, title: str, path: str):
         """
         Constructor for the data visualizer.
 
         Parameters
         ----------
         title : str
                 title of the plot.
+        path : str
+                path to the saving directory of the plot
         """
         if config.jupyter:
             output_notebook()
         else:
-            output_file(f"{title}.html", title=title)
+            print(f"\n the path {path} \n")
```

Comment:
Don't use print - at all - always use logging. But here I don't think we need either